src/test/java/co/rsk/bitcoinj/script/ScriptTest.java

```diff
@@ -706,23 +706,63 @@ public void getSigInsertionIndex_whenScriptWithOnlyOP0AndRedeemScript_shouldRetu
         Sha256Hash hashForSignature = Sha256Hash.of(new byte[]{1});
         BtcECKey signingKey = FEDERATION_KEYS.get(1);
 
-        // This hardcode redeemScript is identified as NoRedeemScriptParser, and is using its findKeyInRedeem implementation
-        // which is returning -1 as default value. This means that the signature insertion index will always be
-        // 0 which is the initial value set when getting the signature insertion index
         int sigInsertionIndex = customRedeemScript.getSigInsertionIndex(hashForSignature, signingKey);
         Assert.assertEquals(0, sigInsertionIndex);
     }
 
     @Test
-    public void isSentToMultiSig_whenFlyoverStandardMultiSig_shouldReturnTrue() {
-        Script redeemScript = RedeemScriptUtils.createStandardRedeemScript(FEDERATION_KEYS);
+    public void isSentToMultiSig_whenNonStandardErpRedeemScriptHardcoded_shouldReturnFalse() {
+        Script nonStandardErpTestnetRedeemScript = new Script(NON_STANDARD_ERP_TESTNET_REDEEM_SCRIPT_SERIALIZED);
+        Assert.assertFalse(nonStandardErpTestnetRedeemScript.isSentToMultiSig());
+    }
 
+    @Test
+    public void isSentToMultiSig_whenFlyoverNonStandardErpRedeemScriptHardcoded_shouldReturnFalse() {
+        Script nonStandardErpTestnetRedeemScript = new Script(NON_STANDARD_ERP_TESTNET_REDEEM_SCRIPT_SERIALIZED);
         Script flyoverRedeemScript = RedeemScriptUtils.createFlyoverRedeemScript(
-            FLYOVER_DERIVATION_HASH,
-            redeemScript
+            FLYOVER_DERIVATION_HASH, nonStandardErpTestnetRedeemScript);
+        Assert.assertFalse(flyoverRedeemScript.isSentToMultiSig());
+    }
+
+    @Test
+    public void isSentToMultiSig_whenEmptyScript_shouldReturnFalse() {
+        Script emptyScript = new ScriptBuilder().build();
+        Assert.assertFalse(emptyScript.isSentToMultiSig());
+    }
+
+    @Test
+    public void isSentToMultiSig_whenInvalidScript_shouldReturnFalse() {
+        Script invalidScript = new Script(new byte[5]);
+        Assert.assertFalse(invalidScript.isSentToMultiSig());
+    }
```

Comment:
```suggestion
    public void isSentToMultiSig_whenEmptyScript_shouldReturnFalse() {
        Script emptyScript = new ScriptBuilder().build();
        assertScriptIsNotMultiSig(emptyScript);
    }

    @Test
    public void isSentToMultiSig_whenInvalidScript_shouldReturnFalse() {
        Script invalidScript = new Script(new byte[5]);
        assertScriptIsNotMultiSig(invalidScript);
    }
    
    private void assertScriptIsNotMultiSig(Script script) {
        Assert.assertFalse(script.isSentToMultiSig());
    }
```

and the same with the other tests. Kind of nit and no big deal, but this gives me more the impression of a "parameterized test". Wdyt?

Reply:
Done